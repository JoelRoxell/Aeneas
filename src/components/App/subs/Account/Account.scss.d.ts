// This file is generated automatically
export const account: string;
export const top: string;
export const corner: string;
export const logo: string;
export const header: string;
export const body: string;
export const view: string;
