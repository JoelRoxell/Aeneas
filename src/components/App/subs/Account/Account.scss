.account {
  display: flex;
  width: 100%;
  height: 100%;
  overflow: auto;
  flex-direction: column;

  .top {
    display: flex;

    .corner {
      width: rem(128);
      height: rem(96);
      background-color: $background;
    }

    .header {
      height: rem(96);
      flex: 1;
    }
  }

  .body {
    display: flex;
    flex: 1;

    .view {
      flex: 1;
      background-color: $background;
      padding: rem(45) rem(64) rem(45) rem(64);
    }
  }

  // @include medium {
  //   flex-direction: column;
  // }

  // @include large {
  //   flex-direction: row;
  // }
}
