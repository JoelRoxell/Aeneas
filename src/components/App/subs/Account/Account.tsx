import * as React from 'react'
import { Switch, Route } from 'react-router'

import * as style from './Account.scss'

function Account() {
  return (
    <div className={style.account}>
      <Switch>
        <Route exact path="/account" render={() => <div>test</div>} />
      </Switch>
    </div>
  )
}

export default Account
