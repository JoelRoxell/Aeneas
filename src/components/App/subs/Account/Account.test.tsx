import * as React from 'react'
import { shallow } from 'enzyme'
import Account from 'components/App/subs/Account'

describe('Account', () => {
  test('render', () => {
    const wrapper = shallow(<Account />)

    expect(wrapper).toMatchSnapshot()
  })
})
