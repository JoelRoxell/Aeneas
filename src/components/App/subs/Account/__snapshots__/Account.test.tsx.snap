// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Account render 1`] = `
<div
  className="account"
>
  <div
    className="top"
  >
    <div
      className="corner"
    />
    <div
      className="header"
    />
  </div>
  <div
    className="body"
  >
    <div
      className="navigation"
    />
    <div
      className="view"
    >
      <Switch>
        <Route
          exact={true}
          path="/account"
          render={[Function]}
        />
      </Switch>
    </div>
  </div>
</div>
`;
